public/data/LakshayVardhan.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Lakshay Vardhan",
+  "bio": "Just a beginner :)",
+  "avatar": "https://github.com/iwantamaste.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "http://github.com/iwantamaste",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "http://twitter.com/VardhanLakshay_",
+      "icon": "twitter"
+    }
+  ]
+}
+  
```

Comment:
GitHub is highlighting that this needs an extra blank line - we can merge and this can be fixed later